apps/dapp-console-api/src/models/wallets.ts

```diff
@@ -0,0 +1,190 @@
+import type { InferInsertModel, InferSelectModel } from 'drizzle-orm'
+import { and, eq } from 'drizzle-orm'
+import {
+  index,
+  jsonb,
+  pgTable,
+  timestamp,
+  uniqueIndex,
+  uuid,
+  varchar,
+} from 'drizzle-orm/pg-core'
+import type { Address } from 'viem'
+
+import type { Database } from '@/db'
+
+import type { Entity } from './entities'
+import { entities } from './entities'
+
+enum WalletState {
+  ACTIVE = 'active',
+  DISABLED = 'disabled',
+  SANCTIONED = 'sanctioned',
+  UNLINKED = 'unlinked',
+}
+
+enum LinkType {
+  PRIVY = 'privy',
+}
+
+type AddressVerification = {}
+
+export const wallets = pgTable(
+  'wallets',
```

Comment:
@nitaliano since I believe this table will primarily be used for dealing with the accounts associated with a user I decided to rename it to wallets. I also added a `linkType` column for determining what provider was used for linking the wallet. I imagine in the future we may have other did providers or even the option for users to manually link their wallets. Let me know what you think of this change.